indexer/packages/v4-protos/src/codegen/klyraprotocol/subaccounts/tx.ts

```
@@ -1,50 +1,50 @@
+export interface MsgClaimYieldsForSubaccountSDKType {
+  /** MsgClaimYieldsForSubaccount is the Msg/ClaimYieldsForSubaccount request type. */
   id?: SubaccountIdSDKType;
 }
 /**
- * MsgClaimYieldForSubaccountResponse is the Msg/ClaimYieldForSubaccount
+ * MsgClaimYieldsForSubaccountResponse is the Msg/ClaimYieldsForSubaccount
  * response type.
  */
 
-export interface MsgClaimYieldForSubaccountResponse {}
+export interface MsgClaimYieldsForSubaccountResponse {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)